bl-kernel/boot/rules/60.plugins.php

```diff
@@ -99,6 +99,18 @@ function buildPlugins()
 
 		// If the plugin is installed insert on the hooks
 		if ($Plugin->installed()) {
+            
+			// If the plugin contains the function "registerHooks" returning a String array, 
+		    	//it will called and if those hooks are not existing they will added
```

Comment:
only some slight readability stuff :) agree with the rest

```suggestion
		    	// it will be called and if those hooks are not existing they will be added
```